CRM/Utils/Check/Component/Schema.php

```diff
@@ -81,8 +81,9 @@ public function checkMissingLogTables() {
     $messages = array();
     $logging = new CRM_Logging_Schema();
     $missingLogTables = $logging->getMissingLogTables();
+    $config = CRM_Core_Config::singleton();
 
-    if ($missingLogTables) {
+    if ($config->logging && $missingLogTables) {
```

Comment:
why not just CRM_Core_Config::singleton()->logging ? @eileenmcnaughton

Reply:
@seamuslee001 I agree that would be better - but I see your ```CRM_Core_Config::singleton()->logging``` and raise you a ```\Civi::setings()->get('logging');```